src/components/tool_tip/_tool_tip.scss

```
@@ -55,6 +55,14 @@
   }
 }
 
+.euiToolTip--displayBlock {
+  display: block;
+
+  .euiToolTipAnchor {
+    display: block;
+  }
+}
```

And then of course this needs to be updated as well
```suggestion
.euiToolTipAnchor--displayBlock {
  display: block;
}
```